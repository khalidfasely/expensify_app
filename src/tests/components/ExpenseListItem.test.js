import React from 'react';
import { shallow } from 'enzyme';
import ExpenseItem from '../../components/ExpenseListItem';
import expenses from '../fixtures/expenses';

test('should render ExpenseItem with data', () => {
    const wrapper = shallow(<ExpenseItem {...expenses[1]} />);
    expect(wrapper).toMatchSnapshot();
});