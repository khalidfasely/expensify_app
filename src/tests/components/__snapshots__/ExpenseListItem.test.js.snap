// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseItem with data 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/2"
  >
    <h3>
      Rent
    </h3>
  </Link>
  <p>
    $1,830.00
    -
    January 5th, 1970
  </p>
</div>
`;
